eg/hygrometer-htu21d.js

```diff
@@ -0,0 +1,16 @@
+var five = require("../lib/johnny-five.js");
+var board = new five.Board();
+
+board.on("ready", function() {
+  var hygrometer = new five.Hygrometer({
+    controller: "HTU21D"
+  });
+
+  hygrometer.on("data", function() {
+    console.log(this.relativeHumidity + " %");
```

Comment:
Instead of being lazy, I went and did some quick reading. I learned that this doesn't produce an absolute humidity. https://www.adafruit.com/datasheets/1899_HTU21D.pdf